pkg/shell/cockpit-server.js

```diff
@@ -393,6 +406,76 @@ PageServer.prototype = {
         $(self.realms).on('notify:Joined.server',
                           $.proxy(self, "update_realms"));
         self.update_realms();
+
+        /* PCP logging
+         */
+
+        /* TODO: Talk to systemd directly.  This is a bit too
+         * cumbersome just for a single service so we cheat and use
+         * cockpitd which has all the hairy code.
+         */
```

Comment:
Why? I'm just about to rewrite this code. It would be nice to start doing things right.